### DISCUSSION TITLE
Tax4Fun2 dependencies

### DISCUSSION THREAD

**Discussion by ironcyborg** [ChiLiubio/microeco#332] [General]
Unfortunately, the links reported in the tutorial (paragraph 2.5.7, Tax4Fun2) for Ref99NR.zip and Ref100NR.zip are broken.
Is there any alternative link, or some way to rebuild those databases locally? Thank you!

**Comment by ChiLiubio** [comment]
Hi. I upload the Ref99NR data in my computer and build a repository. The download link is "https://github.com/ChiLiubio/microeco_extra_data/releases/download/v1.0.0/Tax4Fun2_ReferenceData_v2.zip" or "https://gitee.com/chiliubio/microeco_extra_data/releases/download/v1.0.0/Tax4Fun2_ReferenceData_v2.zip". I will add these in the tutorial. Thanks for your finding!

**Reply by ironcyborg** [reply]
Thank you so much!
